src/dir.cr

```diff
@@ -142,6 +143,11 @@ class Dir
     end
   end
 
+  # Alias for `working_directory`
+  def self.current
+    working_directory
+  end
+
```

Comment:
It would break compactibility, yes, but wouldn't it be better to replace the other function entirely? Instead of creating an alias